flexmeasures/ui/crud/accounts.py

```diff
@@ -0,0 +1,57 @@
+from flask import url_for
+from flask_classful import FlaskView
+from flask_login import current_user
+from flexmeasures.ui.crud.api_wrapper import InternalApi
+
+from flexmeasures.auth.policy import ADMIN_READER_ROLE, ADMIN_ROLE
+from flexmeasures.ui.utils.view_utils import render_flexmeasures_template
+
+
+def get_accounts():
```

Comment:
Missing return type annotation:
```
from future import __annotations__

def get_accounts() -> list[dict]:
```

Because the function name informally suggests `Account` objects will be returned, I find type annotating the return values extra important.

Reply:
That makes sense, implemented!